docs/content/preview/yugabyte-platform/manage-deployments/xcluster-replication/xcluster-replication-setup.md

```diff
@@ -0,0 +1,272 @@
+
+If a YSQL database includes tables considered ineligible for replication, this database cannot be selected in the YugabyteDB Anywhere UI and the database as a whole cannot be replicated.
+
+You can add databases containing colocated tables to the xCluster configuration as long as the underlying database is v2.18.1.0 or later. Colocated tables on the source and target should be created with the same colocation ID if they already exist on both the source and target prior to setup. Refer to [xCluster and colocation](../../../../architecture/docdb-sharding/colocated-tables/#xcluster-and-colocation).
+
+### Set up xCluster replication
+
+To set up replication for a universe, do the following:
+
+1. Navigate to your source universe and select **xCluster Replication**.
+
+1. Click **Configure Replication**.
+
+1. Enter a name for the replication.
+
+1. Select the universe to use as the target.
+
+1. Select the table type, YSQL or YCQL.
+
+1. If you selected YSQL, choose whether to use [transactional atomicity](../#xcluster-configurations).
+
+1. Click **Next: Select Databases** (YSQL) or **Select Tables** (YCQL).
+
+1. Select the databases or tables to be copied to the target for replication.
+
+    For YSQL, select the database(s) with the tables you want in replication. All tables in a selected database are added. Colocated tables require additional conditions. For more information, see [YSQL tables](#ysql-tables).
+
+    For YCQL, you can select individual tables.
+
+1. Click **Validate Table Selection**.
+
+    YugabyteDB Anywhere checks whether or not data needs to be copied to the target for the selected databases and its tables. See [Full copy during xCluster setup](#full-copy-during-xcluster-setup).
+
+1. If data needs to be copied, click **Next: Configure Full Copy**, and select a storage configuration.
+
+    The storage is used to transfer the data to the target database. For information on how to configure storage, see [Configure backup storage](../../../back-up-restore-universes/configure-backup-storage/).
+
+1. Click **Next: Confirm Alert Threshold**.
+
+    If you have [set an alert for replication lag](#set-up-replication-lag-alerts) on the universe, the threshold for alerting is displayed.
+
+1. Click **Confirm and Enable Replication**.
+
+YugabyteDB Anywhere proceeds to set up replication for the universe. How long this takes depends mainly on the amount of data that needs to be copied to the target. You can view the progress of xCluster tasks by navigating to the universe **Tasks** tab.
+
+### Add a database to an existing replication
+
+Note that, although you don't need to create objects on the target when adding tables to an existing replication configuration, for YSQL, if a table is missing on the target, xCluster Replication must copy the entire database. By adding the objects to the target, you can potentially avoid a [full copy](#full-copy-during-xcluster-setup).
```

Comment:
```suggestion
Prepare your database and tables on the source. The source can be empty or have data. If the source has a lot of data, setup will take longer because the [data must be copied in full to the target](https://deploy-preview-23417--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/manage-deployments/xcluster-replication/xcluster-replication-setup/#full-copy-during-xcluster-setup "data must be copied in full to the target") before on-going asynchronous replication starts.

During add database, you don't need to create objects on the target. YugabyteDB Anywhere performs a full copy of the data to be replicated on the source, and automatically creates tables and objects, and restores data on the target from the source.
```